tests/InputFiles/GrMhd/ValenciaDivClean/FishboneMoncriefDisk.yaml

```diff
@@ -87,7 +87,7 @@ Observers:
   ReductionFileName: "GrMhdReductions"
 
 InterpolationTargets:
-  Horizon:
+  InterpToKerrHorizon:
```

Comment:
Sorry for the questions, but I've gotten lost...  I just need reminding of what this actually enables, and what capabilities are coming next...  Does this do anything besides interpolate to the target (i.e. is something written to file?)  Is it being done always or never (I don't see it in the triggers)?    If it does something besides interpolate, how do I control what is observed.   Also I think `KerrHorizon` is sufficient since the `InterpTo` is redundant with `InterpolationTargets` .  Also you should use yaml references to keep the mass and spin of the solution and interpolation target the same...

Reply:
It interpolates and calls obsevers (currently writes the mass flux, but eventually will write other things too.)  You control what is being observed via the InterpolationTargetTag in the metavariables, which specifies the observers. Currently there is no trigger in this input file, so interpolation/observation is done never; a trigger needs to be added to do the interpolation/observation.

Reply:
I can change the name and look at how yaml references work.

Reply:
Okay, thanks for clarifying!